app/models/user.rb

```diff
@@ -88,8 +95,14 @@ class User < ActiveRecord::Base
   after_save :clear_global_notice_if_needed
   after_save :refresh_avatar
   after_save :badge_grant
+  after_save do
+    DiscourseEvent.trigger(:user_verified, user: self) if (self.active_changed? && self.active == true)
```

Comment:
@ZogStriP I thought I read somewhere that I need to keep the brackets if I had multiple conditions .. but after searching again I can't find that information - and it passed the build test in my dev env. :)

Good point on removing true ..